src/Java.Interop.Tools.Cecil/Java.Interop.Tools.Cecil/DirectoryAssemblyResolver.cs

```diff
@@ -114,9 +127,22 @@ public virtual AssemblyDefinition Load (string fileName)
 
 		protected virtual AssemblyDefinition ReadAssembly (string file)
 		{
+			bool haveDebugSymbols = loadDebugSymbols &&
+				(File.Exists (Path.ChangeExtension (file, ".mdb")) ||
+				 File.Exists (Path.ChangeExtension (file, ".pdb")) ||
+				 File.Exists (Path.ChangeExtension (file, ".ppdb")));
 			var reader_parameters = new ReaderParameters () {
-				AssemblyResolver  = this,
-				ReadSymbols       = loadDebugSymbols && (File.Exists(file + ".mdb") || File.Exists(Path.ChangeExtension(file, ".pdb"))),
+				ApplyWindowsRuntimeProjections  = loadReaderParameters.ApplyWindowsRuntimeProjections,
+				AssemblyResolver                = this,
+				MetadataImporterProvider        = loadReaderParameters.MetadataImporterProvider,
+				InMemory                        = loadReaderParameters.InMemory,
+				MetadataResolver                = loadReaderParameters.MetadataResolver,
+				ReadingMode                     = loadReaderParameters.ReadingMode,
+				ReadSymbols                     = haveDebugSymbols,
```

Comment:
We cannot produce pdb on mono without this being fixed. There is PR waiting for approval so any any cecil based tool works properly on mono.

See https://github.com/jbevain/cecil/pull/293